### ISSUE TITLE
Suggestion: A UI for creating pranks and viewing activity/resiliency

### ISSUE THREAD

**Issue by i0natan** [i0natan/node-chaos-monkey#2]
@KromDaniel Sharing a thought - including a UI for quick pranks creation and viewing results can appeal to many. What do you think?

![pranks_log2](https://user-images.githubusercontent.com/8571500/42421458-9467effa-82de-11e8-8c89-8f319f143efd.png)
![new_prank](https://user-images.githubusercontent.com/8571500/42421459-94888238-82de-11e8-8f40-fafcaf58b266.png)

**Comment by TITAN9389** [comment]
![image](https://user-images.githubusercontent.com/27279495/47089639-53260b00-d232-11e8-9311-4c057c0797de.png)

@amaliaman @i0natan just small part if you like the layout.

**Comment by i0natan** [comment]
@TITAN9389 I think it looks great! But I'll let @amaliaman comment - she has much better eye for UX matters... :]

I would just ask if it's possible to include the monkey some place near/within the header

Does the API work fine for you?

**Comment by TITAN9389** [comment]
@i0natan  didn't try the API yet, will try it today and see , for the monkey its the same logo on the main repo page ?

**Comment by i0natan** [comment]
@TITAN9389 Yes, same monkey. You'll probably have to shrink it

**Comment by TITAN9389** [comment]
@i0natan it looks small if include it in header , is it okay ?
![image](https://user-images.githubusercontent.com/27279495/47236822-50701500-d3ee-11e8-8106-76593a77ef13.png)

**Comment by i0natan** [comment]
@amaliaman what do you think?

**Comment by amaliaman** [comment]
Hi, sorry for the delay (kids...)
I think that in the icon should be simpler so it would look coherent in small sizes.
The pranks log looks good!

**Comment by TITAN9389** [comment]
@i0natan  @amaliaman I've Pushed and Open pull request .if you can merge and check the UI View 2 Finished +  responsiveness ( ps. i have some Q tried to connect via skype but non were online.)
![image](https://user-images.githubusercontent.com/27279495/47244369-507c0f00-d406-11e8-9f38-f0b3fc34837f.png)

**Comment by i0natan** [comment]
@TITAN9389 Looks very nice, looking forward to try it. I will be able to be online in Skype in 2-3 hour ,check the UI and bounce with feedback later tonight.

Can you block few hours on Monday or Tuesday for small enhancements?

**Comment by TITAN9389** [comment]
@i0natan okay , yes on Tuesday i guess.

**Comment by TITAN9389** [comment]
@i0natan @amaliaman I've added MobX with Decorators without ejecting CRA , its working fine check it plus i've made some automated npm script so you just need to run `npm run start:app` and it will install UI dependencies & build it then serve it on `localhost:8081`.

**Comment by i0natan** [comment]
@TITAN9389 You rock!

Also including the create prank logic?

**Comment by TITAN9389** [comment]
@i0natan yes everything included(GET,POST,WS). (just UI enhancements left).
